lib/v2/bilibili/utils.js

```diff
@@ -3,7 +3,234 @@ const iframe = (aid, page, bvid) =>
+                            f = y(f, p, u, l, a[d + 15], 16, 530742520),
+                            u = m(u, l = y(l, f, p, u, a[d + 2], 23, -995338651), f, p, a[d + 0], 6, -198630844),
+                            p = m(p, u, l, f, a[d + 7], 10, 1126891415),
+                            f = m(f, p, u, l, a[d + 14], 15, -1416354905),
+                            l = m(l, f, p, u, a[d + 5], 21, -57434055),
+                            u = m(u, l, f, p, a[d + 12], 6, 1700485571),
+                            p = m(p, u, l, f, a[d + 3], 10, -1894986606),
+                            f = m(f, p, u, l, a[d + 10], 15, -1051523),
+                            l = m(l, f, p, u, a[d + 1], 21, -2054922799),
+                            u = m(u, l, f, p, a[d + 8], 6, 1873313359),
+                            p = m(p, u, l, f, a[d + 15], 10, -30611744),
+                            f = m(f, p, u, l, a[d + 6], 15, -1560198380),
+                            l = m(l, f, p, u, a[d + 13], 21, 1309151649),
+                            u = m(u, l, f, p, a[d + 4], 6, -145523070),
+                            p = m(p, u, l, f, a[d + 11], 10, -1120210379),
+                            f = m(f, p, u, l, a[d + 2], 15, 718787259),
+                            l = m(l, f, p, u, a[d + 9], 21, -343485551),
+                            u = u + g >>> 0,
+                            l = l + b >>> 0,
+                            f = f + w >>> 0,
+                            p = p + x >>> 0;
+                    }
+                    return e.endian([u,
+                        l,
+                        f,
+                        p]);
+                };
+            c._ff = function (t, e, n, r, o, i, a) {
+                const s = t + (e & n | ~e & r) + (o >>> 0) + a;
+                return (s << i | s >>> 32 - i) + e;
+            },
+                c._gg = function (t, e, n, r, o, i, a) {
+                    const s = t + (e & r | n & ~r) + (o >>> 0) + a;
+                    return (s << i | s >>> 32 - i) + e;
+                },
+                c._hh = function (t, e, n, r, o, i, a) {
+                    const s = t + (e ^ n ^ r) + (o >>> 0) + a;
+                    return (s << i | s >>> 32 - i) + e;
+                },
+                c._ii = function (t, e, n, r, o, i, a) {
+                    const s = t + (n ^ (e | ~r)) + (o >>> 0) + a;
+                    return (s << i | s >>> 32 - i) + e;
+                },
+                c._blocksize = 16,
+                c._digestsize = 16,
+                t.exports = function (t, n) {
+                    if (null === t) {throw new Error('Illegal argument ' + t);}
+                    const r = e.wordsToBytes(c(t, n));
+                    return n && n.asBytes ? r : n && n.asString ? i.bytesToString(r) : e.bytesToHex(r);
+                };
+        }();
+    }));
+    const ea = function (b, f) {
+        const p = Math.round(Date.now() / 1000);
+        return {
+            w_rid: u(`${b}&wts=${p}${f}`),
```

Comment:
The whole `u` is [MD5](https://github.com/blueimp/JavaScript-MD5/blob/master/js/md5.js) and I believe you can finish `addVerifyInfo` within 10 lines